boshio/boshio.go

```diff
@@ -208,12 +213,15 @@ Retry:
 	var respBytes []byte
 	respBytes, err = ioutil.ReadAll(resp.Body)
 	if err != nil {
-		if err == io.ErrUnexpectedEOF {
-			goto Retry
+		isEOF := err == io.ErrUnexpectedEOF
```

Comment:
It took me awhile to figure out this was just a boolean assignment. Can we make this and the other line any more clear?